metrics-operator/config/rendered/release.yaml

```diff
@@ -21,7 +20,7 @@ spec:
     webhook:
       clientConfig:
         service:
-          name: webhook-service
+          name: metrics-webhook-service
```

Comment:
i don't think we should push this file at all.  it's just the rendered manifest...

Reply:
ups I will remove and add to gitignore